tasks/lib/compass.js

```diff
@@ -201,6 +201,10 @@ exports.init = function (grunt) {
       args.push('--boring');
     }
 
+    if(options.poll) {
```

Comment:
There should be a space after `if` like `if (options.poll)`, to match the current code style.